test/image/compare_pixels_test.js

```diff
@@ -129,34 +119,6 @@ function untestableFilter(mockName) {
     return cond;
 }
 
-/* gl2d pointcloud and other non-regl gl2d mock(s)
- * must be tested first on in order to work;
- * sort them here.
- *
- * gl-shader appears to conflict with regl.
- * We suspect that the lone gl context on CircleCI is
- * having issues with dealing with the two different
- * program binding algorithm.
- *
- * The problem will be solved by switching all our
- * WebGL-based trace types to regl.
- *
- * More info here:
- * https://github.com/plotly/plotly.js/pull/1037
- */
-function sortGl2dMockList(mockList) {
```

Comment:
Looks like we'll have to keep this:

Running

```
npm run test-image
```

off this branch fails at `gl2d_pointcloud-basic`:

![image](https://user-images.githubusercontent.com/6675409/55002199-8ec8bc00-4fac-11e9-8426-127d136ae14e.png)

Reply:
I could reproduce this issue locally! Nice catch :eyes: 

With be336489a93f6600c6ac668f4f7b46b323d99e53, I reintroduced `sortGl2dMockList` and running `npm run test-image` completed without error :tada:

Reply:
![image](https://user-images.githubusercontent.com/6675409/55014567-947ccc80-4fc1-11e9-8988-62b675e21fb8.png)

works for me too :tada: